stdlib/public/core/Array.swift

```diff
@@ -1202,6 +1202,47 @@ extension Array: RangeReplaceableCollection {
     _appendElementAssumeUniqueAndCapacity(oldCount, newElement: newElement)
     _endMutation()
   }
+  
+  /// Adds the elements of a collection to the end of the array.
+  ///
+  /// Use this method to append the elements of a collection to the end of this
+  /// array. This example appends the elements of a `Range<Int>` instance
+  /// to an array of integers.
+  ///
+  ///     var numbers = [1, 2, 3, 4, 5]
+  ///     numbers.append(contentsOf: 10...15)
+  ///     print(numbers)
+  ///     // Prints "[1, 2, 3, 4, 5, 10, 11, 12, 13, 14, 15]"
+  ///
+  /// - Parameter newElements: The elements to append to the array.
+  ///
+  /// - Complexity: O(*m*) on average, where *m* is the length of
+  ///   `newElements`, over many calls to `append(contentsOf:)` on the same
+  ///   array.
+  @inlinable @_alwaysEmitIntoClient
```

Comment:
This is a little misleading: `@_alwaysEmitIntoClient` already implies inlinability, but `@inlinable` carries an expectation that it will export an ABI symbol. I recommend just using aeic, like we do in most similar cases in the stdlib.

```suggestion
  @_alwaysEmitIntoClient
```

Reply:
I'll fix this once I figure out the SIL test failure :) I had just @inlinable originally but Steve correctly pointed out that would make the Sequence-taking one explode when back-deployed if it didn't get inlined.

Reply:
Done!